pkg/controller/cloud/nodecontroller.go

```diff
@@ -55,6 +65,13 @@ const (
 
 	// The amount of time the nodecontroller should sleep between retrying NodeStatus updates
 	retrySleepTime = 20 * time.Millisecond
+
+	//Taint denoting that a node needs to be processed by external cloudprovider
+	CloudTaintKey = "ExternalCloudProvider"
```

Comment:
I like the idea of namespacing. @luxas any suggestions for the actual taint?

WDYT about this?
`external.cloudprovider.kubernetes.io="true":NoSchedule`